cif_file_ingester/parse_cif_pmg.py

```diff
@@ -59,9 +61,16 @@ def parse_cif(f):
     ##### Use pymatgen to quickly obtain some information #####
     try:
         structure = Structure.from_file(f)
-    except:
-        print(f, 'is not interpretable by pymatgen')
-        return None
+    except Exception:
+        # Try with ASE instead
+        try:
+            ase_res = ase.io.read(f)
+            assert ase_res
```

Comment:
Can we cover the None return case more explicitly?  What does it mean when ase returns None?

Reply:
It's not necessarily returning None, but ase can "succeed" at reading a file but have no data associated with it. I figured it would be better to have that cause a failure during reading than have it cause a failure later on when nonexistent data is accessed. I can change this check, though, if there's a more preferable way to handle it.

Reply:
I'm just looking for a little more information in the logs than a generic assertion error.  We expose the ingest logs to users on citrination, so they're are the first line of debugging.

Reply:
Oh, I misunderstood. I'll add a different exception there.